llvm/test/Other/dump-before-after.ll

```diff
@@ -0,0 +1,57 @@
+; RUN: mkdir -p %t/logs
+; RUN: rm -rf %t/logs
+
+; Basic dump before and after a single module pass
+
+; RUN: opt %s -disable-output -passes='no-op-module' -ir-dump-directory %t/logs -print-after=no-op-module -print-before=no-op-module
+; RUN: find %t/logs -type f -print | sort | FileCheck %s --check-prefix=SINGLE-PASS
```

Comment:
Instead of calling `sort`, we can take advantage of the `DAG` directive: https://www.llvm.org/docs/CommandGuide/FileCheck.html#the-check-dag-directive

Reply:
You can also add something like `--implicit-check-not='.ll'` to be sure that we aren't dumping extra files

Reply:
I was using `DAG` before, but in this case I'm checking the the log files are numbered in the order they run. I don't want to assert what the numbers should be, just that they are ordered correctly. 

There is a comment about this below.

Reply:
> I will say using sort won't work if there are more than 9 passes (10-foo is alphabetically before 8-bar), so this test could be a little fragile.

I think you can try `sort -n` to sort the files based on the first numeric value.